MaxMind.GeoIP2.UnitTests/DatabaseReaderTests.cs

```diff
@@ -95,7 +95,7 @@ public void AnonymousIP_ValidResponse()
                 Assert.False(response.IsPublicProxy);
                 Assert.False(response.IsTorExitNode);
                 Assert.Equal(ipAddress, response.IPAddress);
-                Assert.Equal("1.2.0.0/16", response.Network.ToString());
+                Assert.Equal("1.2.0.0/16", response.Network?.ToString());
```

Comment:
Why ? instead of ! on these ones?

Reply:
I figured an assert error about `"1.2.0.0/16"` not being equal to null was nicer than a null pointer error if this ever did fail. It'd probably be even nicer to assert that `response.Network` is not null first, but that seemed like too much effort given that it doesn't seem that likely and it wasn't a new issue.